tests/conftest.py

```diff
@@ -3590,7 +3590,7 @@ def bobby_columnar_table_multi_batch(empty_data_context):
                 "kwargs": {
                     "column": "fare_amount",
                     "min_value": 228.94,
-                    "max_value": 2990.05,
+                    "max_value": 2990.06,
```

Comment:
Why the change here?

Reply:
@cdkini This is the key part of this PR -- rounding was handled incorrectly and also the builder was run twice instead of once.